ci/matlab.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This is a basic workflow to help you get started with MATLAB actions
+# For more information, see: https://github.com/matlab-actions/overview
+
+name: MATLAB
```

Comment:
Should we rename it to "MATLAB CI"? Some templates have named their workflows as [such](https://github.com/actions/starter-workflows/blob/1d039e1607c98d9b9a705102315b43896ad10633/ci/node.js.yml#L4). This would only change the workflow name and not the template name (it depends on the properties file).

Reply:
I think MATLAB alone is fine, but it's up to you and the team. Not sure if MATLAB CI raises any legal flags.

Reply:
Cool, we'll leave it then